requirements.txt

```diff
@@ -1,11 +1,10 @@
 diffusers==0.29.2
 accelerate
 transformers
-xformers
+xformers; sys_platform != 'darwin'
```

Comment:
xformers can't be installed on Mac: https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/11396

As far as I understand, this dependency is not imported explicitly in the code and the model can do inference on cpu on Mac, just very slow